draft-bastian-jose-dvs.md

```diff
@@ -92,16 +92,34 @@ A primary motivation for this work is to enable HMAC signature validation from i
 
 {::boilerplate bcp14-tagged}
 
-# Terminology
-
-The draft uses "JSON Web Signature", "JOSE Header", "JWS Signature", "JWS Signing Input" as defined by {{RFC7515}}.
```

Comment:
Why is this part being removed? Terminology section is very common in IETF standards